internal/test/test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package test
+
+import (
+	"flag"
+	"testing"
+)
+
+func init() {
+	testing.Init()
+	flag.Parse()
+}
+
+// IsTestRun will return whether or not we're operating within the context of a
+// test.
```

Comment:
It looks to me like this function actually checks whether some verbose test flag is set. Is that correct? I like the functionality, but I think the function is confusingly named and documented. Maybe `IsVerboseTestRun`?

Also, is the intent to override the behavior of tests configured to disable event logging if a verbose test run is requested? If so, would it make it impossible to run a verbose test run without all the events?

Reply:
pushed an amended commit.. is that better?

Reply:
This seems reasonable for now, but I noticed that Go 1.21 will include a function for this: https://github.com/golang/go/commit/7f38067acb738c43d870400dd648662d31456f5f. Maybe we could add a TODO to this comment to replace it with this new function when available?

Reply:
happy to.  ty!